ArtifactRegistry/src/V1beta2/resources/artifact_registry_rest_client_config.php

```diff
@@ -243,51 +243,5 @@
                 ],
             ],
         ],
-        'google.longrunning.Operations' => [
```

Comment:
@noahdietz I have been seeing this block being removed from a few of our clients... is this intended behavior or did we do something to make this block unnecessary?